src/webp/vp8.rs

```diff
@@ -1701,7 +1705,11 @@ fn predict_4x4(ws: &mut [u8], stride: usize, modes: &[IntraMode], resdata: &[i32
                 IntraMode::HU => predict_bhupred(ws, x0, y0, stride),
             }
 
-            add_residue(ws, rb, y0, x0, stride);
+            // Create a [i32; 16] array for add_residue by copying the
+            // slice from resdata into rb (slices do not work).
+            let mut rb = [0i32; 16];
+            rb.copy_from_slice(&resdata[i * 16..i * 16 + 16]);
```

Comment:
The change here left aliased the `rb` a few lines above which is now unused. Could you remove it?